target/linux/ar71xx/image/generic.mk

```diff
@@ -801,6 +801,18 @@ define Device/jwap230
 endef
 TARGET_DEVICES += jwap230
 
+define Device/k2t
+  DEVICE_TITLE := Phicomm K2T A1/A2/A3 board (16MB flash)
+  DEVICE_PACKAGES := kmod-usb-core kmod-usb2 kmod-usb-storage kmod-ath10k ath10k-firmware-qca9888
```

Comment:
and the usb-related packages here...
BTW for a device with USB support, only usb host driver should be included here. Device drivers like kmod-usb-storage shouldn't be here.